testsuite/scripts/src/test/java/org/wildfly/scripts/test/DomainScriptTestCase.java

```diff
@@ -15,22 +20,42 @@
 import org.jboss.as.test.shared.TestSuiteEnvironment;
 import org.jboss.dmr.ModelNode;
 import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 import org.wildfly.common.test.ServerHelper;
 
 /**
  * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
  */
+@RunWith(Parameterized.class)
 public class DomainScriptTestCase extends ScriptTestCase {
 
     private static final Function<ModelControllerClient, Boolean> HOST_CONTROLLER_CHECK = ServerHelper::isDomainRunning;
 
+    @Parameterized.Parameter
+    public Map<String, String> env;
     public DomainScriptTestCase() {
         super("domain");
     }
 
+    @Parameterized.Parameters
+    public static Collection<Object> data() {
+        final Collection<Object> result = new ArrayList<>(2);
+        result.add(Collections.emptyMap());
+        result.add(Collections.singletonMap("SECMGR", "true"));
+        return result;
+    }
+
     @Override
     void testScript(final ScriptProcess script) throws InterruptedException, TimeoutException, IOException {
-        script.start(HOST_CONTROLLER_CHECK, ServerHelper.DEFAULT_SERVER_JAVA_OPTS);
+        boolean isSecurityManagerEnabled = Boolean.parseBoolean(System.getProperty("security.manager"));
```

Comment:
IMO we should skip this check and just always test with the security manager enabled in one run. Currently we don't run security manager tests on Windows so we'd lose that testing.